### ISSUE TITLE
more of a UI text than translation it seems

### ISSUE THREAD

**Issue by godbout** [ploi-deploy/ploi-translations#11]
`OPCache` should be `OPcache`. if there's a place to do a PR for those kinds of text, let me know. thanks!

**Comment by Cannonb4ll** [comment]
Good call, there is currently no translation yet for that, since we are still in the process of adding the translation keys to every part of the panel.

Once it is added we'll reply to this issue.

**Comment by godbout** [comment]
👌🏼️👍🏼️
